### ISSUE TITLE
Qt 6.8 not supported

### ISSUE THREAD

**Issue by xuzhongxing** [i486/VxKex#47]
If my program is compiled with Qt 6.7, it can run with vxkex on Win 7. If it is compiled with Qt 6.8, windows plugin cannot be loaded.

**Comment by Kiki79250CoC** [comment]
Maybe you should start by providing more details than just "it doesn't work".

What is broken?
Do you have any error message?
If it's about a missing function, which one?
Which VxKex version have you used?
Have you attempted to workaround this? If yes, what have you tried?

The purpose here to simplify developer's life is to make issue reports that are the most complete possible. Just saying it doesn't work won't help anyone. The developer is not a divinity and thus cannot guess what issue you're facing and what you've tried before creating this """"""issue report"""""" that contains no useful information.

Please, next time, consider doing the bare bare minimum which is **mentioning what error you've got**. Developers should not go to fish for details, those information should be present from the very beginning so the developer can process your issue quicker and more efficiently.

**Comment by xuzhongxing** [comment]
My program is compiled with Qt 6.8.1. Qt 6.8.1 is built on Windows 7. The vxkex version is 1.1.2.1428. 

The error message I got is:
qt.qpa.plugin: Could not load the Qt platform plugin "windows" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: direct2d, minimal, offscreen, windows.

**Comment by dotexe1337** [comment]
VxKex dotexe version 20250101
Added functions:
- UIAutomationCore.dll - UiaRaiseNotificationEvent
Now Qt 6.8.x software can run! =D
Happy new years, let's keep extending in 2025
https://dotexe.cf/dl/VxKex_20250101.zip
![image](https://github.com/user-attachments/assets/40690de2-8037-41e2-895b-0ebd2dac14e4)

**Comment by xuzhongxing** [comment]
Where can I find the repo of this version?

**Comment by xuzhongxing** [comment]
Yes, version 20250101 works for Qt 6.8.x.

**Comment by dotexe1337** [comment]
> Where can I find the repo of this version?

github.com/dotexe1337/VxKex

**Comment by xuzhongxing** [comment]
Thank you. I'm wondering what will be the relation with this repo.

**Comment by CommonLoon102** [comment]
> > Where can I find the repo of this version?
> 
> github.com/dotexe1337/VxKex

Why it isn't forked from i486's repo?
There are some forks already, it would be nice if all the additions would be merged to a base repo, so that it could have every new kernel function and functionality.

**Comment by dotexe1337** [comment]
> Why it isn't forked from i486's repo?

It is. i486's repo is the base VxKex version from vxiiduu, which is what mine is based on.

**Comment by CommonLoon102** [comment]
> > Why it isn't forked from i486's repo?
> 
> It is. i486's repo is the base VxKex version from vxiiduu, which is what mine is based on.

But you haven't properly forked it on GitHub, just copied the source, therefore it is not possible to open pull requests.

**Comment by dotexe1337** [comment]
@CommonLoon102 
> But you haven't properly forked it on GitHub, just copied the source, therefore it is not possible to open pull requests.

I'm not sure, maybe my Git client isn't configured for that. I mostly just use Git as file storage to put my code on, and haven't really touched most of the fancy features.

**Comment by CommonLoon102** [comment]
> @CommonLoon102
> 
> > But you haven't properly forked it on GitHub, just copied the source, therefore it is not possible to open pull requests.
> 
> I'm not sure, maybe my Git client isn't configured for that. I mostly just use Git as file storage to put my code on, and haven't really touched most of the fancy features.

1. Delete your current VxKex repo
2. Click the Fork button at https://github.com/i486/VxKex
3. Clone the newly created repo to your PC
4. Copy and overwrite files at local repo with your new/modified files
5. Push

**Comment by i486** [comment]
Yeah, I've already been thinking about making an 'experimental' branch where we can test things out and devs to work together on PRs. Before I do that, I'll make an 'announcement' thread to hopefully get everyone on board. I'll do that sooner or later.

**Comment by YuZhouRen86** [comment]
> VxKex dotexe version 20250101 Added functions:
> 
>     * UIAutomationCore.dll - UiaRaiseNotificationEvent
>       Now Qt 6.8.x software can run! =D
>       Happy new years, let's keep extending in 2025
>       https://dotexe.cf/dl/VxKex_20250101.zip
>       ![image](https://private-user-images.githubusercontent.com/41847745/399596158-40690de2-8037-41e2-895b-0ebd2dac14e4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYxNTI1MDcsIm5iZiI6MTczNjE1MjIwNywicGF0aCI6Ii80MTg0Nzc0NS8zOTk1OTYxNTgtNDA2OTBkZTItODAzNy00MWUyLTg5NWItMGViZDJkYWMxNGU0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTA2VDA4MzAwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYzOTYxYTM3MWU0NzZlYjg2ZTZkZWJkYTQzZGY4N2Q1NTYwYjNiMDA0ZTNhY2U5MjhjM2M3MjJkMjM2ZTA4YWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.nrXRRIzrAgZfW5wBULFinWW2O5oUMFnnqeJXt7oFYYU)

Good! I have made change to my project too, just declare the func and let it always return S_OK. Kate editor and Qt Designer latest version works fine.
![图片](https://github.com/user-attachments/assets/d4a60fd9-e503-47fb-b275-8278de5ea73f)
![图片](https://github.com/user-attachments/assets/9c88b8c2-c7c9-40cc-8779-15b387d1bc0b)

**Comment by Komprogue** [comment]
> Yeah, I've already been thinking about making an 'experimental' branch where we can test things out and devs to work together on PRs. Before I do that, I'll make an 'announcement' thread to hopefully get everyone on board. I'll do that sooner or later.

So any chances [these](https://github.com/YuZhouRen86/VxKex-NEXT) [three](https://github.com/dotexe1337/VxKex) [forks](https://github.com/Aqua-Vision/MgKex) (or, at least, changes they made) would be ever united into one build?

**Comment by Neustradamus** [comment]
I have done tickets to have all improvements in one place.

Linked to:
- https://github.com/i486/VxKex/issues/101
- https://github.com/i486/VxKex/issues/103
- https://github.com/i486/VxKex/issues/104
- Not mine: https://github.com/i486/VxKex/issues/36

**Comment by i486** [comment]
We are closing this issue as part of a cleanup of long-standing inactive reports.

If this issue is still relevant and reproducible with the latest version, feel free to reopen or create a new issue with updated details.
